### ISSUE TITLE
Bug: User names behind Woka

### ISSUE THREAD

**Issue by anbraten** [thecodingmachine/workadventure#1738]
User names are behind the woka on Chrome based browser:

![Screenshot from 2022-01-19 11-21-58](https://user-images.githubusercontent.com/6918444/150111666-09ede762-e4d9-41fe-bb95-a8e5303625fb.png)

Expected:
- Fix code for chrome based browsers to show name above wokas head
- Maybe draw name after drawing woka, so it would at least be drawn on top of the woka.

**Comment by anbraten** [comment]
This issue is caused by some protection in Brave and other privacy extensions which change the canvas api to protect from fingerprinting. 

https://github.com/photonstorm/phaser/issues/5625

**Comment by anbraten** [comment]
Maybe a fix for Brave users not being aware of it would be to change the z-index of the name to be above the woka itself.

**Comment by moufmouf** [comment]
Yes, it's really Phaser / Brave related and the Phaser team won't fix the issue: https://github.com/photonstorm/phaser/issues/5625#issuecomment-810070692

Indeed, we could try to change the z-index of the name to make the issue more bearable for Brave users :+1:

**Comment by gparant** [comment]
@moufmouf @nolway, we could add background black color on the text and have better legibility? What do you think?
